assets/src/components/amp-support/errors.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * To render error information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Error data.
+ * @return {JSX.Element|null} HTML markup for error data.
+ */
+export function Errors( { data: errors } ) {
+	if ( 'object' !== typeof errors ) {
```

Comment:
The `propTypes` say that `errors` is an array. In this case, I think that the check should be more specific:

```suggestion
	if ( Array.isArray( errors ) ) {
```